asdf/compression.py

```diff
@@ -188,6 +188,9 @@ def get_compressed_size(data, compression, block_size=1 << 16):
     compression : str
         The type of compression to use.
 
+    block_size : int, optional
+        The size of blocks (in raw data) to process at a time.
```

Comment:
This comment isn't quite clear to me: I think what it's supposed to say is something like "Input data will be chunked into blocks of this size (in bytes) before compression"